moped-etl/prefect/moped_test_create_flow.py

```diff
@@ -47,24 +48,34 @@
 
 @task(name="Slug branch name")
 def slug_branch_name(basename):
-    short_tls_basename = basename[0:27]
-    elb_basename = basename[0:32]
-    underscore_basename = basename.replace("-", "_")
-    database = re.search("^[\d_]*(.*)", underscore_basename).group(
-        1
-    )  # remove leading numbers
-    internal_number_free_underscore_basename = "".join(
-        [i for i in database if not i.isdigit()]
+
+    graphql_endpoint = (
+        basename.replace("_", "-").replace(".", "-").strip("0123456789-_")
+    )
```

Comment:
I almost want to disable `black` formatting on these lines so they all are a single line and to have them each carefully hand indented so you get columns of similar operations. 🤷🏻‍♂️